src/transactions/TransactionFrameBase.cpp

```diff
@@ -24,4 +24,10 @@ TransactionFrameBase::makeTransactionFromWire(Hash const& networkID,
         abort();
     }
 }
+
+Hash
+TransactionFrameBase::txHashForFlooding(TransactionFrameBasePtr const& tx)
```

Comment:
this does not seem to be the right place for this helper: `TransactionFrameBase` should not decide what overlay does